packages/app-store/_components/AppCard.tsx

```diff
@@ -116,7 +116,7 @@ export default function AppCard({
         {app?.isInstalled && switchChecked && <hr className="border-subtle" />}
 
         {app?.isInstalled && switchChecked ? (
-          app.isSetupAlready ? (
+          app.isSetupAlready === undefined || app.isSetupAlready ? (
```

Comment:
So for all other apps that don't require setup we keeping undefined, explicit true should display fields and explicit false should display button to go to setup.
![image](https://github.com/calcom/cal.com/assets/4461358/8526f60c-816c-4b5c-93f7-90597abe9de7)